config/cms.php

```diff
@@ -380,5 +380,19 @@
     */
 
     'restrictBaseDir' => true,
+    
+    /*
+     |--------------------------------------------------------------------------
+     | Component default partial fallback
+     |--------------------------------------------------------------------------
+     |
+     | Enable using default theme partial for Component fallback where exists.
+     |
+     | If is enabled, especially each snippet not need to have own theme
+     | override, but uses default one.
+     |
+     */
+    
+    'enableComponentThemeDefaultPartialFallback' => false,
```

Comment:
@Rike-cz This config name is too large. Maybe something shorter?
```suggestion
    'defaultAliasPartialFallback' => false,
```